src/scss/vault.scss

```diff
@@ -414,14 +414,6 @@ app-root {
       border-top-color: themed("borderColor");
     }
 
-    button {
-      margin-right: 10px;
-
-      &:last-child {
-        margin-right: 0;
-      }
-    }
-
```

Comment:
This is used for buttons when editing an item, and we'll need to keep it for now.

Reply:
oh, I missed that. I can have a second look at this later tonight.

Reply:
ah i know what happened there. removed that margin to fix the "My Vault"/"Sends" (now changed to buttons) having a gap, but missed that this was needed in the footer of the edit view for items.